sbin/init/init.c

```diff
@@ -358,7 +358,7 @@ main(int argc, char *argv[])
 #ifdef __DEBUG_CHERI_TRAP_DURING_INIT__
```

Comment:
This is some interesting code...

Reply:
Hmm not sure I intended to commit that as part of https://github.com/CTSRD-CHERI/cheribsd/commit/7f98bae3b7d0add10e3cbb7f1796f9408992b5d5... Pretty sure that was just some test code to ensure the cheribuild test script exits early on an init failure. Should be removed instead of updated IMO.